ndex2/client.py

```diff
@@ -1035,6 +1506,18 @@ def grant_network_to_user_by_username(self, username, network_id, permission):
         """
         Grants permission to network for the given user name
 
+	.. code-block:: python
+		
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		username = "INSERT USERNAME HERE"
+		network_id = 'INSERT NETWORK UUID HERE'
+		my_ndex.grant_network_to_user_by_username(username, network_id, permission = "READ")
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client